pyodide_build/buildall.py

```diff
@@ -72,10 +72,15 @@ def build(self, outputdir: Path, args) -> None:
                     stderr=subprocess.STDOUT,
                 )
 
-        with open(self.pkgdir / "build.log", "r") as f:
-            shutil.copyfileobj(f, sys.stdout)
+        try:
+            p.check_returncode()
+        except subprocess.CalledProcessError:
+            print(f"Error building {self.name}. Printing build logs.")
 
-        p.check_returncode()
+            with open(self.pkgdir / "build.log", "r") as f:
+                shutil.copyfileobj(f, sys.stdout)
+
+            raise
```

Comment:
Do we actually need to print the full traceback here, as opposed to just printing the logs +  error message and doing a `sys.exit(1)`. The nested exceptions with no relevant information can be confusing to users.